### PR TITLE
Add https to 'npm run serve'

### PR DESCRIPTION
**Pull request by txsmith**

### CODE REVIEW

**Comment by eanker** [comment]
The backend integration seems to be broken, by this pr.
I'll investigate this issue.

File: package.json [hunk 1]
```diff
@@ -19,8 +19,8 @@
   },
   "scripts": {
    "build": "node ./node_modules/gulp/bin/gulp.js",
-   "serve": "node ./node_modules/polyserve/bin/polyserve -p 8080 --proxy-path api/v1 --proxy-target http://localhost:8443/",
-   "serve-build": "node ./node_modules/polyserve/bin/polyserve build/unbundled/ -p 8080 --proxy-path api/v1 --proxy-target http://localhost:8443/"
+   "serve": "node ./node_modules/polyserve/bin/polyserve -p 8080 --proxy-path api/v1 --proxy-target http://localhost:8443/ -P h2",
+   "serve-build": "node ./node_modules/polyserve/bin/polyserve build/unbundled/ -p 8080 --proxy-path api/v1 --proxy-target http://localhost:8443/ -P h2"
```
Comment (eanker) [inline package.json:24]: Why don't we do -P h2 at serve-build?

Comment (TimvdLippe) [reply package.json:24]: You do, it's at the end of the line 😉

**Review by TimvdLippe — APPROVED** [review]

**Comment by eanker** [comment]
It works.

**Pull request merged**